admin/bin/dev-shell

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -eu
+
+TOPLEVEL=$(git rev-parse --show-toplevel)
+
+#!/bin/bash
+# shellcheck disable=SC2086
```

Comment:
Lint rule disabled should include a justifying comment.

Reply:
# we ignore SC2086 because ${DOCKER_BUILD_ARGUMENTS:-} is intended to
# be evaluated into multiple strings, not a single argument.

Is that reasonable ? Or are there other ways to do the same ?